runtime/doc/news.txt

```diff
@@ -148,6 +148,10 @@ VIMSCRIPT
   treats BIN, STR and FIXSTR as separate types. Any of these is returned as a
   string if possible, or a |blob| if the value contained embedded NUL:s.
 
+BUILD SYSTEM
```

Comment:
The sections are in alphabetical order, this should go after "API". Could also just name it `BUILD` or maybe `CORE` or `INTERNAL`

Reply:
it is not `INTERNAL` . This note is for third-party distros (one of the main consumers of release notes)